packages/manager/apps/container/src/container/legacy/server-sidebar/universe/DedicatedSidebar.tsx

```diff
@@ -393,15 +393,6 @@ export default function DedicatedSidebar() {
       });
     }
 
-    if (feature.ip) {
-      menu.push({
-        id: 'dedicated-ip',
-        label: t('sidebar_ip_short'),
-        icon: getIcon('ovh-font ovh-font-ip'),
-        href: navigation.getURL('dedicated', '#/ip'),
-        routeMatcher: new RegExp('/ip(/|$)'),
-      });
-    }
```

Comment:
**question:** Why `IP` is removed in a PR concerning `vRack Services`?

Reply:
It was a rebase mistake the ip was moved inside network by a ticket made by David few weeks ago